packages/amplify-cli/src/__tests__/extensions/amplify-helpers/constants.test.ts

```diff
@@ -0,0 +1,12 @@
+import { amplifyCLIConstants } from '../../../extensions/amplify-helpers/constants';
+
+describe('constants', () => {
+  it('should freeze values', () => {
+    expect(amplifyCLIConstants.AmplifyCLIDirName).toBe('amplify');
+    expect(() => {
+      // @ts-ignore
```

Comment:
It is possible to make this more specific instead of suppressing all errors?

Reply:
I tried `//// TS2540` comment to make it more specific, but it did not work out unfortunately and that made compile time error instead of runtime error. It would be great if we can do that.

e.g. how that did not work:
https://www.typescriptlang.org/play?#code/MYewdgzgLgBAZiEMC8MDeAoGMDWBTATwC4YByANwEMAbAVz1IwF8ZKIZRIoBuDDBEADp8BFGVoAHACaUoeKaW4wA9KuUwAKgGUATAFYALAAYgA

Reply:
If you just need TS to stop complaining about reassigning a const value, I think you can do `(amplifyCLIConstants as any).AmplifyCLIDirName = 'test'`

Reply:
@edwardfoyle That's true! I just updated this PR to use `as any` in overwriting test. Thank you :)